userspace/libsinsp/k8s_handler.cpp

```diff
@@ -94,7 +96,8 @@ k8s_handler::k8s_handler(const std::string& id,
 		g_logger.log(std::string("K8s (" + m_id + ") creating handler for " +
 							 uri(m_url).to_string(false) + m_path), sinsp_logger::SEV_DEBUG);
 		m_handler = std::make_shared<handler_t>(*this, m_id, m_url, m_path, m_http_version,
-											 m_timeout_ms, m_ssl, m_bt, !m_blocking_socket, m_blocking_socket);
+											 m_timeout_ms, m_ssl, m_bt, !m_blocking_socket, m_blocking_socket,
+											 524288u, true, data_max_mb, data_chunk_wait_us);
```

Comment:
524288? Are we trading one magic constant for another?

Reply:
Thank you. This should be addressed in efa93793dfa05e1ae26562ec57847bf7ec700525.